### ISSUE TITLE
Vingester 2.6 and 2.6.1 NDI issue

### ISSUE THREAD

**Issue by me-vlad** [rse/vingester#53]
Vingester ver. 2.6 and 2.6.1 on Windows 10 x86_64 - missing NDI output and preview,  vingester-browser-preload.js load errors in a console (please see a screenshot below).
Vingester 2.5 works as expected. 

![image](https://user-images.githubusercontent.com/222185/133932798-d754a26f-1dfa-4cd2-8ee3-46b60afaa5c2.png)

**Comment by rse** [comment]
Interesting, under macOS I do not get any errors in the console, but under Windows 10 I see the same problems than you. I can at least repeat the problem. I've to investigate what causes this. The difference between 2.5 and 2.6 is mainly the newer Electron 14. Perhaps it has something to do with it...

**Comment by rse** [comment]
The problem is caused by Electron 14 (at least under Windows), because when I downgrade to 13.3.0 the problem is gone. Hmmm.... strange, seems to be either a bug in Electron 14 or an incompatibility perhaps caused by Vingester. I've to dive deeper...

**Comment by rse** [comment]
Up to the latest Electron 13.3.0 it works fine, with 14.0.0/14.0.1 is breaks under Windows. The root cause seems to be some network issues inside Electron, as on the console I also see:

```
[17936:0919/180803.407:ERROR:network_service_instance_impl.cc(333)] Network service crashed, restarting service.
```

The "network_service_instance_impl.cc" is from Chromium inside Electron. And it is fully unrelated to NDI, as the problem under Windows 10 also exists with the Vingester frameless output mode.

**Comment by rse** [comment]
Ok, Electron 13.4.0 still works just fine, Electron 14.0.0/14.0.1 breaks, so I'll downgrade to Electron 13.4.0 for Vingester 2.6.2.

**Comment by rse** [comment]
Now fixed with 2.6.2
